Assets/MixedRealityToolkit/_Core/Utilities/PlatformUtility.cs

```diff
@@ -37,18 +32,13 @@ public static bool IsPlatformSupported(this RuntimePlatform runtimePlatform, Sup
                 case RuntimePlatform.LinuxEditor:
                     return (platform & SupportedPlatforms.LinuxStandalone) != 0;
                 default:
-                    return false;
+                    return true;
```

Comment:
not sure we want to default to true here... if we do not support a specific platform, we should not say we do.

Reply:
I don't understand. This is the catch all for everything. If we don't specify one of the other's then it's by default everything.

Reply:
If we don't specify one of the supported platforms, then it'll either be none or everything. returning true here supports the everything flag without having to specify additional platforms.